README.md

```diff
@@ -7,27 +7,7 @@ Carbon is a library of reusable [React](https://facebook.github.io/react/) compo
 See the [demo and documentation site](https://carbon.sage.com/) for live examples, or see [create-carbon-app](https://github.com/sage/create-carbon-app) to get started.
 
 ## Documentation
-
-### Getting Started
-
-* [Setting up your environment](docs/guides/setting-up-your-environment.md)
-* [Getting started](docs/guides/getting-started.md)
-
-### Guides
-
-* [Assets](docs/guides/assets.md)
-* [Immutable](docs/guides/immutable.md)
-* [Validations](docs/guides/validations.md)
-* [Decorators](docs/guides/decorators.md)
-* [Services](docs/guides/services.md)
-* [Integrating Carbon with other UI](docs/guides/integrating-with-other-ui.md)
-* [Testing changes to Carbon in your project](docs/guides/installing-unreleased-changes.md)
-* [Testing process and tools](cypress/README.md)
-* [Releasing Carbon](docs/guides/releasing.md)
-
-#### Testing
-
-As Carbon is just React components, we support any test suite you prefer to use. However we also provide out-of-the-box test tooling through [Carbon Factory](https://github.com/sage/carbon-factory).
+The Carbon getting started and coding guides have been removed as they we're getting increasingly out-of-date. Apologies for the inconvenience this may cause. We plan to revisit these and publish new documentation in due course, in the meanwhile please feel free to raise any questions to the maintainers as a [New Issue](https://github.com/Sage/carbon/issues).
```

Comment:
```suggestion
The Carbon getting started and coding guides have been removed as they were getting increasingly out-of-date. Apologies for the inconvenience this may cause. We plan to revisit these and publish new documentation in due course, in the meanwhile please feel free to raise any questions to the maintainers as a [New Issue](https://github.com/Sage/carbon/issues).
```
I think this is beyond the remit of carbon. I think its best that we say nothing at all here.